marimo/_cli/sandbox.py

```diff
@@ -69,6 +69,9 @@ def _read_pyproject(script: str) -> Dict[str, Any] | None:
 
 
 def prompt_run_in_sandbox(name: str | None) -> bool:
+    if GLOBAL_SETTINGS.MANAGE_SCRIPT_METADATA:
```

Comment:
We should make sure people install the version of marimo they are running and not the latest version